Makefile

```diff
@@ -542,6 +544,10 @@ upload-preloaded-images-tar: out/minikube # Upload the preloaded images for olde
 push-storage-provisioner-image: storage-provisioner-image ## Push storage-provisioner docker image using gcloud
 	gcloud docker -- push $(STORAGE_PROVISIONER_IMAGE)
 
+.PHONY: push-kic-base-image
+push-kic-base-image: kic-base-image ## Push kic-base docker image using gcloud
+	gcloud docker -- push $(KIC_BASE_IMAGE)
```

Comment:
we should first check to see there is no image with same name and tag, if there is not, we can push, if there is we will echo "image already exists"

Reply:
:+1:  Added 1 line above:

It will try to pull an image. I can add `timeout 5` before docker pull if preferred.

I don't see any smarter option than pulling the image, as `docker manifest` is still experimental.